src/core/pointcloud/qgspointcloudrenderer.h

```diff
@@ -292,6 +344,27 @@ class CORE_EXPORT QgsPointCloudRenderer
       y = context.offset().y() + context.scale().y() * iy;
     }
 
+    /**
+     * Copies common point cloud properties (such as point size and screen error) to the \a destination renderer.
+     */
+    void copyCommonProperties( QgsPointCloudRenderer *destination ) const;
+
+    /**
+     * Restores common renderer properties (such as point size and screen error) from the
+     * specified DOM \a element.
+     *
+     * \see saveCommonProperties()
+     */
+    void restoreCommonProperties( const QDomElement &element, const QgsReadWriteContext &context );
+
+    /**
+     * Saves common renderer properties (such as point size and screen error) to the
+     * specified DOM \a element.
+     *
+     * \see restoreCommonProperties()
+     */
+    void saveCommonProperties( QDomElement &element, const QgsReadWriteContext &context ) const;
+
```

Comment:
isn't enough these to be protected?

Reply:
They already are -- I think the diff is just misleading here.